src/src/Controller/CollaborationController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class CollaborationController extends AbstractController
+{
+    #[Route('/collaboration', name: 'app_collaboration')]
+    public function index(): Response
+    {
+        return $this->render('collaboration/index.html.twig', [
+            'controller_name' => 'CollaborationController',
```

Comment:
This variable is generated by the framework as demo code. It should be removed as you don't need it.